content/prompts/ai-development/responsible-ai-checker.md

```diff
@@ -0,0 +1,50 @@
+---
+authors:
```

Comment:
Author is not required for new content. Author will fetched from git info during build time. I will also check the contribution instructions and templates if there is an error in the documentation